framework/src/main/java/org/checkerframework/framework/type/TypeFromExpressionVisitor.java

```diff
@@ -270,7 +270,10 @@ public AnnotatedTypeMirror visitMemberSelect(MemberSelectTree tree, AnnotatedTyp
         } else {
             // tree must be a field access, so get the type of the expression, and then call
             // asMemberOf.
-            AnnotatedTypeMirror t = f.getAnnotatedType(tree.getExpression());
+            // As long as everuseflow is true, always have flow-refinement for receiver.
```

Comment:
The use of `everuseflow` here is confusing. Can you add a clearer reference?
Maybe in general try to clarify this motivation.

Reply:
added more explanation to the modification.